### PR TITLE
Disable scientific notation for tjson floats

### PR DESCRIPTION
**Pull request by noisersup**
# Description

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [ ] I ensured that PR title is good enough for the changelog.
* [ ] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [ ] I marked all done items in this checklist.

### CODE REVIEW

File: internal/handlers/tigris/tjson/double.go [hunk 1]
```diff
@@ -50,10 +53,16 @@ func (d *doubleType) UnmarshalJSON(data []byte) error {
...
-		return nil, lazyerrors.Error(err)
-	}
+	// This will default to precision of 6
+	//res := []byte(fmt.Sprintf("%f", float64(*d)))
+
+	// get precision of the value to preserve it
+	dStr := strconv.FormatFloat(float64(*d), 'f', -1, 64)
+	precision := len(dStr) - strings.Index(dStr, ".") - 1
+
+	res := []byte(fmt.Sprintf("%."+strconv.Itoa(precision)+"f", float64(*d)))
```
Comment (AlekSi) [inline internal/handlers/tigris/tjson/double.go:63]:
```suggestion
	res := []byte(dStr)
```

Comment (noisersup) [reply internal/handlers/tigris/tjson/double.go:63]: That will force the default precision somewhere I believe

Comment (AlekSi) [reply internal/handlers/tigris/tjson/double.go:63]: Precision is the third argument of `FormatFloat` – `-1` there

Comment (noisersup) [reply internal/handlers/tigris/tjson/double.go:63]: I didn't remember the exact issue so reproduced it:
```
                                        expected: "179769313486231570814527423731704356798070567525844996598917476803157260780028538760589558632766878171540458953514382464234321326889464182768467546703537516986049910576551282076245490090389328944075868508455133942304583236903222948165808559332123348274797826204144723168738177180919299881250404026184124858368.000000"
                                        actual  : "179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```
I guess I'll need to understand the function a bit more

**Comment by AlekSi** [comment]
That should be fixed by #1916

**Pull request closed without merging**